index.src.html

```diff
@@ -2685,6 +2695,9 @@
   // Default display dimensions match visibleRect.
   [EnforceRange] unsigned long displayWidth;
   [EnforceRange] unsigned long displayHeight;
+
+  // Defaults to Rec709 or SRGB for planar and interleaved formats respectively.
```

Comment:
For YUV and RGB formats respectively.

Reply:
I've deleted this comment now, but see how I've implemented the "Pick Color Space" algo (choose sRGB if it's an RGB format, otherwise choose rec709).